packages/block/src/block.ts

```diff
@@ -242,6 +252,15 @@ export class Block {
    * @param blockchain - additionally validate against an @ethereumjs/blockchain instance
    */
   async validateUncles(blockchain: Blockchain): Promise<void> {
+    /*
+      The rules of uncles are the following:
+      Uncle Header is not in canonical chain
+      Uncle Header has a parentHash which points to the Canonical Chain. This parentHash is within the last 7 blocks.
+      Uncle Header is not included in the canonical chain.
+      Header has at most 2 uncles
+      Header does not count an uncle twice.
```

Comment:
Not a blocker, but please keep an eye on being more consistent with things like upper/lower case spellings (minimally consistency within a single comment block 😜, better throughout the library, optimally unified across all libraries) (e.g. `Canonical Chain`), punctuation marks yes/no, stuff like that.

Reply:
Ah yes you are right, this looks sloppy 😅